docs/numeric-types/Snippets.re

```diff
@@ -0,0 +1,75 @@
+// #region counter
+module Counter = {
+  [@react.component]
+  let make = () => {
+    let (counter, setCounter) = React.useState(() => 0);
+
+    <div
+      style={ReactDOMStyle.make(
```

Comment:
ftr this is changing soon to `ReactDOM.Style` in reasonreact. Maybe worth opening an issue to remember or something.

Reply:
Create new issue: https://github.com/melange-re/melange-for-react-devs/issues/3#issue-1977099173